vnext/Microsoft.ReactNative/JsiApi.idl

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.ReactNative
+{
+  interface IJsiRuntime;
+
+  delegate void JsiDataHandler(UInt8[] data);
+
+  [webhosthidden]
+  interface IJsiBuffer
+  {
+    UInt32 Size { get; };
+    void GetData(JsiDataHandler handler);
+  };
+
+  enum JsiValueType
+  {
+    Undefined,
+    Null,
+    Boolean,
+    Number,
+    Symbol,
+    String,
+    Object,
+  };
+
+  [webhosthidden]
+  struct JsiValueData
+  {
+    JsiValueType Kind;
+    UInt64 Value;
```

Comment:
>UInt64 Value; [](start = 4, length = 13)

Could you add a comment explaining what this value is?

Reply:
Yes, I will do.

---
In reply to: [451846390](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r451846390) [](ancestors = 451846390)

Reply:
@vmorozov: I'm assuming thse are handles. Does making these 64bit cause potential tearing/threading issue on 32-bit or should the size be platform specific?

---
In reply to: [454518335](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r454518335) [](ancestors = 454518335,451846390)